packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -222,10 +286,29 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
     });
     const addressBottomText = isAddressWithLabel ? addressLabelComponent : null;
 
+    const getBottomText = () => {
+        if (addressError) {
+            return (
+                <InputError message={addressError.message} button={getValidationButtonProps()} />
+            );
+        }
+
+        if (addressChecksumInfo) {
+            return (
+                <InputChecksumInfo
+                    message="TR_CHECKSUM_CONVERSION_INFO"
+                    tooltipContent="Info about checksummed address"
+                />
+            );
+        }
+
+        return addressBottomText;
+    };
+
     return (
         <Container>
             <Input
-                inputState={inputState}
+                inputState={inputState()}
```

Comment:
nit:
```suggestion
                getInputState={inputState()}
```

Reply:
I'm not sure I understand why this rename is needed

Reply:
you are right, i wanted to comment it like this

```suggestion
                inputState={getInputState()}
```

Reply:
ah there is conflict in names.

However, `inputState` is a function so it should have name which stress that it is a function and not a variable

Reply:
makes sense, I will think of another name

Reply:
Renamed to getInputErrorState()